### PR TITLE
Create APIReference.md

### PR DESCRIPTION
**Pull request by naynaly10**
Kevin, we need a dedicated repo for documentation.

https://docusaurus.io/ has a nice template.

### CODE REVIEW

File: plans/APIReference.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+OpenAI API Key
+
+<br/>
+
+#### Example Code
+
+```javascript
+import { Samantha } from 'socialAGI'
+
+const samantha = new Samantha('sk-123456789')
```
Comment (kafischer) [inline plans/APIReference.md:69]: i think we probably want samantha to be like a persona that is initialized somehow vs her own thing

**Comment by kafischer** [comment]
could we just put docs in `/docs`?

File: plans/APIReference.md (hunk 1 above)
Comment (naynaly10) [reply plans/APIReference.md:69]: I agree. We need you to focus in on Samantha's code. There must be a way for creators to create more powerful Samanthas with the API.

**Comment by naynaly10** [comment]
> could we just put docs in `/docs` instead of separate repo?

Sounds good. :) But beautiful design is important for a great experience for the community developers and perhaps we could use Sphinx to convert to JSON to Markdown.

**Comment by kafischer** [comment]
Yeah you're totally right we need beautiful docs - I think I had forgotten what Spinx looked like 😂

**Pull request closed without merging**